R/queries.R

```diff
@@ -62,10 +62,9 @@ get_perfomance_graphs <- function(workout_ids, every_n = 5) {
 #' get_all_workouts()
 #' }
 #'
-get_all_workouts <- function(userid = Sys.getenv("PELOTON_USERID")) {
-  # TODO pagination  in API ?
-  if (userid == "") stop("Provide a userid or set an environmental variable `PELOTON_USERID`", call. = FALSE)
-  workouts <- peloton_api(path = glue::glue("api/user/{userid}/workouts"))
+get_all_workouts <- function(userid = Sys.getenv("PELOTON_USERID"), num_workouts = 20) {
+  if (userid == "") stop("Provide a userid or set an environmental variable `PELOTON_USERID`", ca, ll. = FALSE)
```

Comment:
```suggestion
  if (userid == "") stop("Provide a userid or set an environmental variable `PELOTON_USERID`", call. = FALSE)
```
looks like there is a typo here

Reply:
You are right - must have messed that up after I checked it locally.  Thanks for that catch.